### PR TITLE
Register gabivlg.is-a.dev

### PR DESCRIPTION
**Pull request by GabiVlg**
Register gabivlg.is-a.dev with CNAME record pointing to gabivlg.is-a.dev 10800 IN CNAME domain.par.clever-cloud.com..

### CODE REVIEW

**Comment by GabiVlg** [comment]
I want to change me the cname to 
CNAME | gabivlg | www.gabivlg.is-a.dev

**Comment by GabiVlg** [comment]
i'm using now cloudflare

File: domains/gabivlg.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "GabiVlg",
+           "email": "gabivoinea111111@yahoo.com",
+           "discord": "368732292408934403"
+        },
+    
+        "record": {
+            "CNAME": "gabivlg.is-a.dev 10800 in cname domain.par.clever-cloud.com."
```
Comment (orxngc) [inline domains/gabivlg.json:9]: Firstly, your CNAME has to actually point somewhere, not to the domain you want. Secondly, your CNAME cannot have anything except the domain name itself, e.g. `orangc.xyz`, not `orangc.xyz uwu owo in cname`.

**Comment by GabiVlg** [comment]
gabivlg CNAME www.is-a.dev.cdn.cloudflare.net

File: domains/gabivlg.json (hunk 1 above)
Comment (GabiVlg) [reply domains/gabivlg.json:9]: gabivlg CNAME www.gabivlg.is-a.dev.cdn.cloudflare.net

**Comment by CuteDog5695** [comment]
> gabivlg CNAME [www.is-a.dev.cdn.cloudflare.net](http://www.is-a.dev.cdn.cloudflare.net)

Site isn't accessible though

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**